specs/AutomationProvider.md

```diff
@@ -0,0 +1,55 @@
+# Background
+WebView has two hosting modes, windowed (which uses the ICoreWebView2Controller) and visual (which uses the ICoreWebView2CompositionController). Accessibility for the windowed WebView is able to walk the HWND tree to know where to place the WebView in the accessibility tree. For visual hosting, accessibility is not able to know where within the accessibility tree to place the WebView. This results in a WebView being a sibling to the rest of the app content.
+
+In this document we describe the updated API. We'd appreciate your feedback.
+
+
+# Description
+To give apps using the ICoreWebView2CompositionController more control over how the WebView is positioned in the accessibility tree, we are adding new APIs to let the app retrieve the automation provider for the WebView. This let's the app return the automation provider as part of its accessibility tree.
```

Comment:
Feels like this spec needs a few background links. Is the app responsible for fetching the automation provider for the hwnd? What can someone do what that provider? What type is really hiding behind the IUnkown?

Reply:
Agreed, a little more info on the accessibility would be super helpful.